robottelo/cli/import_.py

```diff
@@ -405,6 +407,31 @@ def content_view_with_tr_data(cls, options=None):
         return (result, transition_data)
 
     @classmethod
+    def content_host_with_tr_data(cls, options=None):
+        """Import Content Hosts (from spacewalk-report system-profiles).
+
+        :returns:
+        :raises:
```

Comment:
Bah...this piece of code is not supposed to be here (it's supposed for another commit)..let me delete that